<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCalendarWidget</class>
 <widget class="QWidget" name="CCalendarWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #999999;
    height: 2px;
    margin: 0px 0;
    left: 12px; right: 12px;
}
QSlider::handle:horizontal {
    border: 0px;
    background-color: rgb(255, 164, 31);
    border-radius:6px;
    height:12px;
    width: 12px;
    margin: -5px -5px -5px -5px;
 }

QSlider::sub-page:horizontal{
    background-color: rgb(255, 164, 31);
}
QSlider::add-page:horizontal{
    background-color: rgb(223, 223, 223);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="CCalendarWnd" name="calendarWidget">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>350</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>时</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderHour">
       <property name="maximum">
        <number>23</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelHourVal">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>分</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderMinute">
       <property name="maximum">
        <number>59</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMinuteVal">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>秒  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderSecond">
       <property name="maximum">
        <number>59</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSecondVal">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSubmit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CCalendarWnd</class>
   <extends>QCalendarWidget</extends>
   <header>components/ccalendarwnd.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
